src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,7 +1,9 @@
 package mate.academy
 
 fun greetUsers() {
-    // implement function
+    println("Please enter a name (or press Enter to quit):" )
+    val name = readlnOrNull();
```

Comment:
As a improvement you may use elvis operator for default value in case if readlnOrNull() returns null.
Just FYI, not mistake